samples/distributed_assignments/assignment-name.codeval

```diff
@@ -0,0 +1,47 @@
+Z assignment-helper.zip
```

Comment:
since these are examples, it might be good to put a comment as to what a verb means before each new verb used.

Reply:
Added comments in new [PR](https://github.com/SJSU-CS-systems-group/CodEval/pull/12)